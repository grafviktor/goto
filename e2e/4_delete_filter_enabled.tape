Set Shell bash
Enter
Type "gg -f temp"
Enter
Sleep 100ms
Type "n"
Sleep 100ms
Type "network host"
Down
Ctrl+u
Type "127.0.0.1"
Down
Type "network host description"
Ctrl+S
Sleep 100ms
Type "c" # create "network host (1)"
Sleep 100ms
Type "c" # create "network host (2)"
Sleep 100ms
Type "c" # create "network host (3)"
Sleep 100ms
Type "c" # create "network host (4)"
Sleep 100ms
Type "c" # create "network host (5)"
Sleep 100ms
Type "/" # enter filter mode
Type "3" # enter "3", so only "network host (3)" is visible
Enter    # Accept filter result
Type "d" # ask delete "network host (3)"
Type "y" # say "yes" to "network host (3)"
Sleep 100ms
Type "q"