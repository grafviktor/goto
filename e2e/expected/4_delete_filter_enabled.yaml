- host:
    title: network host
    description: network host description
    address: 127.0.0.1
- host:
    title: network host (1)
    description: network host description
    address: 127.0.0.1
- host:
    title: network host (2)
    description: network host description
    address: 127.0.0.1
- host:
    title: network host (4)
    description: network host description
    address: 127.0.0.1
- host:
    title: network host (5)
    description: network host description
    address: 127.0.0.1
